### PR TITLE
Improving documentation of Set-AzVMADDomainExtension

### PR DESCRIPTION
**Pull request by Sandido**
## Description

This cmdlet was flagged as having a success rate below our desired threshold. I edited the documentation to correct the example it uses to show how to use the cmdlet. This was to improve the success rate of the cmdlet. The example previously did not use all required parameters. 

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/1143
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Compute/Compute/Extension/ADDomain/SetAzureVMADDomainExtensionCommand.cs [hunk 1]
```diff
@@ -50,7 +50,7 @@ public class SetAzureVMADDomainExtensionCommand : SetAzureVMExtensionBaseCmdlet
         [Parameter(
              Mandatory = false,
              ValueFromPipelineByPropertyName = true,
-             HelpMessage = "Join Options")]
+             HelpMessage = "Specifies the join option.For join options see[JoinOptions](https://docs.microsoft.com/windows/desktop/api/lmjoin/nf-lmjoin-netjoindomain)")]
```
Comment (BethanyZhou) [inline src/Compute/Compute/Extension/ADDomain/SetAzureVMADDomainExtensionCommand.cs:53]:
```suggestion
             HelpMessage = "Specifies the join option. For join options see [JoinOptions](https://docs.microsoft.com/windows/desktop/api/lmjoin/nf-lmjoin-netjoindomain)")]
```

File: src/Compute/Compute/ChangeLog.md [hunk 2]
```diff
@@ -20,6 +20,7 @@
 
 -->
 ## Upcoming Release
+* Edited the documentation for the cmdlet `Set-AzVMADDomainExtension` to ensure the example is accurate.
```
Comment (BethanyZhou) [inline src/Compute/Compute/ChangeLog.md:23]: I think we don't need to publish new version for a doc improvement. What's your thought? Could you please remove this change log?

Comment (BethanyZhou) [reply src/Compute/Compute/ChangeLog.md:23]: Updated docs will be available to customer in April 5th. No matter a new version of module whether publishes.

File: src/Compute/Compute/help/Set-AzVMADDomainExtension.md [hunk 3]
```diff
@@ -28,7 +28,12 @@ This extension lets your virtual machine join a domain.
 
 ### Example 1
 ```powershell
-Set-AzVMADDomainExtension -ResourceGroupName "ResourceGroup11" -VMName "VirtualMachine07" -DomainName "abc.com"
+# Create a Resource Group and Virtual Machine before this.
+$extensionName = "extensionName";
+$virtualMachineName = "vmName";
+$resourceGroupName = "resourceGroupName";
+$domainName = "domain.com";
+Set-AzVMADDomainExtension -ResourceGroupName $resourceGroupName -VMName $virtualMachineName -Name $extensionName -DomainName $domainName;
```
Comment (BethanyZhou) [inline src/Compute/Compute/help/Set-AzVMADDomainExtension.md:36]:
```suggestion
$extensionName = "extensionName"
$virtualMachineName = "vmName"
$resourceGroupName = "resourceGroupName"
$domainName = "domain.com"
Set-AzVMADDomainExtension -ResourceGroupName $resourceGroupName -VMName $virtualMachineName -Name $extensionName -DomainName $domainName
```
PowerShell script doesn't need to end with semicolon

**Review by BethanyZhou — CHANGES_REQUESTED** [review]

File: src/Compute/Compute/ChangeLog.md (hunk 2 above)
Comment (Sandido) [reply src/Compute/Compute/ChangeLog.md:23]: I'll remove the changelog update.

File: src/Compute/Compute/help/Set-AzVMADDomainExtension.md (hunk 3 above)
Comment (Sandido) [reply src/Compute/Compute/help/Set-AzVMADDomainExtension.md:36]: @BethanyZhou , I know it doesn't need to, but I think it makes the lines more clear to the reader. 
Is there an official Best Practice about not using Semicolons?

Comment (BethanyZhou) [reply src/Compute/Compute/help/Set-AzVMADDomainExtension.md:36]: No. But in our conventions, we don't add semicolon in the end of line.

**Comment by BethanyZhou** [comment]
Hi @Sandido , could you see my first suggestion and resolve conflicts?

**Review by BethanyZhou — APPROVED** [review]
LGTM

**Review by BethanyZhou — APPROVED** [review]

**Pull request merged**